Documentation/CMake.md

```diff
@@ -59,21 +59,34 @@ endif()
 ## Add an entry point
 
 You must include a source file in your test executable target with a
-`@main` entry point. The following example uses the SwiftPM entry point:
+`@main` entry point. The example main below requires the experimental
+`Extern` feature. The declaration of `swt_abiv0_getEntryPoint` could
+also be written in a C header file with its own `module.modulemap`.
```

Comment:
Since the function is exported, a developer could also use `dlsym()` or `GetProcAddress()` or equivalent, so a note about that is worthwhile (the implementation does not need to be provided here.)

Reply:
Though actually, one can never go wrong with a POSIX.1-2024 reference https://pubs.opengroup.org/onlinepubs/9799919799/functions/dlsym.html